src/platform/packages/shared/kbn-esql-validation-autocomplete/scripts/generate_function_definitions.ts

```diff
@@ -609,6 +660,38 @@ const replaceParamName = (str: string) => {
   }
 };
 
+const enrichGrouping = (
```

Comment:
Is there a reason you decided to go with a custom enrich function instead of using the existing enrichments system (`functionEnrichments`)?

Reply:
Honestly I didnt notice it. I moved the bucket overwrite there [839f606](https://github.com/elastic/kibana/pull/210513/commits/839f606aa4c0755ccf7aa60ecd7d34a42adef40c)

I kept the enrichGrouping though as I want to apply the ` supportedOptions: ['by'],` to all of them